app/models/order.rb

```diff
@@ -1,19 +1,34 @@
 class Order < ApplicationRecord
 
   enum status: [:pending, :packaged, :shipped, :cancelled]
-  #validates_numericality_of :status
 
   belongs_to :user
   has_many :order_items
   has_many :items, through: :order_items
 
 
   def item_quantity
-    items.count
+    order_items.sum :quantity
```

Comment:
This adjustment was made because previously on the count of unqiue items were coming back. Now the quantity on the order is taken into account.